packages/base/base64-image.gts

```diff
@@ -204,58 +216,32 @@ class ImageSizeField extends FieldDef {
   static edit = class Edit extends Component<typeof this> {
     <template>
       <div class='radio-group' data-test-radio-group={{@fieldName}}>
-        <label for='{{this.radioGroup}}_actual'>
-          Actual
-          <input
-            data-test-actual-size-input
-            type='radio'
-            {{RadioInitializer @model 'actual'}}
-            id='{{this.radioGroup}}_actual'
-            name='{{this.radioGroup}}'
-            checked={{eq @model 'actual'}}
-            {{on 'change' (pick 'target.value' (fn @set 'actual'))}}
-          />
-        </label>
-        <label for='{{this.radioGroup}}_contain'>
-          Contain
-          <input
-            data-test-contain-size-input
-            type='radio'
-            {{RadioInitializer @model 'contain'}}
-            id='{{this.radioGroup}}_contain'
-            name='{{this.radioGroup}}'
-            checked={{eq @model 'contain'}}
-            {{on 'change' (pick 'target.value' (fn @set 'contain'))}}
-          />
-        </label>
-        <label for='{{this.radioGroup}}_cover'>
-          Cover
-          <input
-            data-test-cover-size-input
-            type='radio'
-            {{RadioInitializer @model 'cover'}}
-            id='{{this.radioGroup}}_cover'
-            name='{{this.radioGroup}}'
-            checked={{eq @model 'cover'}}
-            {{on 'change' (pick 'target.value' (fn @set 'cover'))}}
-          />
-        </label>
+        <RadioInput
+          @items={{this.items}}
+          @groupDescription='Image Size Field'
+          name='{{this.radioGroup}}'
+          @checkedId={{this.checkedId}}
+          @hideBorder={{true}}
+          as |item|
+        >
+          <item.component @onChange={{fn @set item.data.id}}>
+            {{item.data.text}}
+          </item.component>
+        </RadioInput>
       </div>
-      <style>
-        .radio-group {
-          display: flex;
-          justify-content: space-between;
-        }
-      </style>
+      <style></style>
     </template>
 
     private radioGroup = `__cardstack_img_size${groupNumber++}__`;
-    constructor(owner: unknown, args: any) {
-      super(owner, args);
-      // initializes to 'actual'
-      if (!this.args.model) {
-        this.args.set('actual');
```

Comment:
This was triggering auto-save. Which is why there was an inconsistency in the behaviour of the new cards. They were "saved" sometimes upon being created.